.github/workflows/aws-ecs-deploy.yml

```diff
@@ -0,0 +1,70 @@
+name: Deploy to Amazon ECS
+
+on:
+  push:
+    branches:
+      - "feat/aws-ecs-deployment"
```

Comment:
Is this the branch or main?

Reply:
For now its for a this specific branch, later will make it for develop and staging

Reply:
Ok